Makefile

```diff
@@ -42,6 +42,9 @@ PORT ?= 4000
 # Access dev server or locally built web app files
 DEV_SERVER ?= false
 
+# Port for dev server
```

Comment:
I found a copy-paste artifact 🙂
(Btw it may be helpful to put the possible choices in the comments here)

Reply:
Good idea! The comments are now updated!